src/main/msp/msp_box.c

```diff
@@ -87,10 +87,10 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { .boxId = BOXBEEPGPSCOUNT, .boxName = "GPS BEEP SATELLITE COUNT", .permanentId = 37 },
 //    { .boxId = BOX3DONASWITCH, .boxName = "3D ON A SWITCH", .permanentId = 38 }, (removed)
     { .boxId = BOXVTXPITMODE, .boxName = "VTX PIT MODE", .permanentId = 39 },
-    { .boxId = BOXUSER1, .boxName = "USER1", .permanentId = 40 },
-    { .boxId = BOXUSER2, .boxName = "USER2", .permanentId = 41 },
-    { .boxId = BOXUSER3, .boxName = "USER3", .permanentId = 42 },
-    { .boxId = BOXUSER4, .boxName = "USER4", .permanentId = 43 },
+    { .boxId = BOXUSER1, .boxName = BOX_USER_1_NAME, .permanentId = 40 },
```

Comment:
```suggestion
    { .boxId = BOXUSER1, .boxName = BOX_USER_1_NAME, .permanentId = 40 }, // may be overridden by modeActivationConfig
```

Reply:
@ledvinap It seems to me that it's better to do this immediately in the structure declaration of `static const box_t boxes[]`